docs/api/initialize.md

```diff
@@ -1,14 +1,22 @@
 ### initialize method
 
 ```js
-FlowRouter.initialize();
+FlowRouter.initialize(options);
 ```
  - Returns {*void*}
```

Comment:
@macrozone need to list and define all arguments above `return`, as example take a [look here](https://github.com/VeliovGroup/flow-router/blob/master/docs/api/getParam.md)